<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AI_Settings</class>
 <widget class="QDialog" name="AI_Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AI_Settings</string>
  </property>
  <widget class="QWidget" name="background" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>1080</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/Resource2/UI_Design/AI_Settings.png);</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="challenger_name">
   <property name="geometry">
    <rect>
     <x>806</x>
     <y>424</y>
     <width>275</width>
     <height>44</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
    </font>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit{
	background:transparent;
	border: 1px solid rgba(0,0,0,0.3);
	padding:5px;
    border-radius:8px;
}</string>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="lineWrapColumnOrWidth">
    <number>0</number>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="tabStopWidth">
    <number>80</number>
   </property>
   <property name="cursorWidth">
    <number>1</number>
   </property>
   <property name="placeholderText">
    <string>您可以在这留名</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="diff2_normal">
   <property name="geometry">
    <rect>
     <x>933</x>
     <y>603</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton{
    spacing: 0px;
}

QRadioButton::indicator {
    width: 31px;
    height: 31px;
}
QRadioButton::indicator:checked {
  image: url(:/Resource2/UI_Design/AI_settings_checked.png);
}
QRadioButton::indicator:unchecked {
    image: url(:/Resource/UI_Design/RadioButton.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="diff3_hard">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1037</x>
     <y>603</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton{
    spacing: 0px;
}

QRadioButton::indicator {
    width: 31px;
    height: 31px;
}
QRadioButton::indicator:checked {
  image: url(:/Resource2/UI_Design/AI_settings_checked.png);
}
QRadioButton::indicator:unchecked {
    image: url(:/Resource/UI_Design/RadioButton.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="diff1_easy">
   <property name="geometry">
    <rect>
     <x>829</x>
     <y>603</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton{
    spacing: 0px;
}

QRadioButton::indicator {
    width: 31px;
    height: 31px;
}
QRadioButton::indicator:checked {
  image: url(:/Resource2/UI_Design/AI_settings_checked.png);
}
QRadioButton::indicator:unchecked {
    image: url(:/Resource/UI_Design/RadioButton.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="isFirst">
   <property name="geometry">
    <rect>
     <x>903</x>
     <y>691</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="back_button">
   <property name="geometry">
    <rect>
     <x>1715</x>
     <y>81</y>
     <width>86</width>
     <height>50</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ClosedHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Resource/UI_Design/#Block.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../../Resource/Gobang.qrc">
     <normaloff>:/Resource/UI_Design/Fun4_BackButton.png</normaloff>:/Resource/UI_Design/Fun4_BackButton.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>86</width>
     <height>50</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../../Resource/Gobang.qrc"/>
  <include location="../../../Resource/Gobang2.qrc"/>
 </resources>
 <connections/>
</ui>
